tests/providers/openlineage/extractors/test_manager.py

```diff
@@ -118,6 +120,7 @@ def test_convert_to_ol_dataset_from_table_with_columns_and_owners():
                 OwnershipDatasetFacetOwners(name="user:<jane@company.com>", type=""),
             ]
         ),
+        "description": DocumentationDatasetFacet(description="test description"),
```

Comment:
You have `description` here and `documentation` in code.

Reply:
Repaired, thanks !